nilearn/image/tests/test_resampling.py

```diff
@@ -145,6 +145,42 @@ def test_downsample():
                                        rot_img.get_data()[:x, :y, :z, ...])
 
 
+def test_resampling_fill_value():
+    """ Test resampling with a non-zero fill value
+    """
+    prng = np.random.RandomState(10)
+
+    data_3d = prng.rand(1, 4, 4)
+    data_4d = prng.rand(1, 4, 4, 3)
+
+    angle = np.pi/4
+    rot = rotation(0, angle)
+
+    # Try a few different fill values
+    for data in [data_3d, data_4d]:
+        for val in (-3.75, 2.24, 0):
```

Comment:
```suggestion
        for val in (-3.75, 0):
```
With 3D and 4D data, and three values, that's 6 calls in the test. I would like to keep it down: I am not sure what different code paths it tests, and it increases computational cost.

Reply:
I would have like to have

- one test specifying a value, e.g. -3.75 here
- one test that doesn't specify a value, and checks we get 0, i.e. default behavior remains the same